src/consensus/cfund.cpp

```diff
@@ -37,61 +41,113 @@ void CFund::SetScriptForPaymentRequestVote(CScript &script, uint256 prequesthash
 bool CFund::FindProposal(string propstr, CFund::CProposal &proposal)
 {
 
-    return pblocktree->ReadProposalIndex(uint256S("0x"+propstr), proposal);
+    return CFund::FindProposal(uint256S("0x"+propstr), proposal);
```

Comment:
Just checking - There is no way this can activate until the next release

Reply:
Activation of the consensus changes related to CFund is triggered by a different soft fork.